TramsDataApi.Test/Integration/ApplyToBecomeIntegrationTests.cs

```diff
@@ -142,17 +165,26 @@ public async Task CanGetApplicationByApplicationId()
                 SchoolConsultationStakeholders = true,
                 SchoolConsultationStakeholdersConsult = _randomGenerator.NextString(2, 10),
                 SchoolSupportGrantFundsPaidTo = _randomGenerator.NextString(2, 10),
-                SchoolDeclarationSignedByName = _randomGenerator.NextString(2, 10)
-            };
+                SchoolDeclarationSignedByName = _randomGenerator.NextString(2, 10),
+                SchoolAdEqualitiesImpactAssessmentDetails = _randomGenerator.NextString(2,10),
+                SchoolLeases = new List<A2BSchoolLease>() { lease },
+                SchoolLoans = new List<A2BSchoolLoan>() { loan }
+           };
+
+            //var applyingSchool = Builder<A2BApplicationApplyingSchool>
```

Comment:
I think we should remove this comment out for now, if / when we come to refactor the tests, we can look at alternative approaches (prob not using Fizzware.NBuilder) to building the model

Reply:
yes - removed